public/components/GeoPlot.vue

```diff
@@ -456,18 +573,31 @@ export default Vue.extend({
       this.renderer = new BatchQuadOverlayRenderer();
       this.overlay.setRenderer(this.renderer);
       // convert this.areas to quads in normalized space and add to overlay layer
-      const normalizedQuads = this.areaToQuads();
+      if (!this.bucketFeatures.length) {
+        this.currentState = this.tileState;
+      } else {
+        this.currentState = this.clusterState;
+        this.map.on(lumo.ZOOM_END, this.onZoom);
+      }
+      if (!this.bucketFeatures.length && !this.areas.length) {
+        this.map.add(this.overlay);
```

Comment:
Why adding an overlay if there is no data?

Reply:
The overlay still needs to be added. I should just move that above the if statement.